account_avatax_oca/models/account_move.py

```diff
@@ -477,26 +477,7 @@ def _get_avatax_amount(self, qty=None):
         adjusted to compute line amount instead of unit price.
         """
         self.ensure_one()
-        base_line = self
-        move = base_line.move_id
-        sign = -1 if move.is_inbound() else 1
-        quantity = qty or base_line.quantity
-        base_amount = base_line.price_unit * quantity
-        if base_line.currency_id:
-            price_unit_foreign_curr = (
-                sign * base_amount * (1 - (base_line.discount / 100.0))
-            )
-            price_unit_comp_curr = base_line.currency_id._convert(
-                price_unit_foreign_curr,
-                move.company_id.currency_id,
-                move.company_id,
-                move.date,
-            )
-        else:
-            price_unit_comp_curr = (
-                sign * base_amount * (1 - (base_line.discount / 100.0))
-            )
-        return -price_unit_comp_curr
+        return -self.amount_currency
```

Comment:
I'm unsure with this, as it is removing the discount handling logic.
There is the risk that discounts are not processed properly after this change.

Reply:
@dreispt 
![discount](https://github.com/OCA/account-fiscal-rule/assets/4667326/dc087e97-b2f7-4d70-858d-2d28ac79d646)
![avatax1](https://github.com/OCA/account-fiscal-rule/assets/4667326/fd28209d-9b2c-413f-b776-40b7f4f38c00)

The amount_currency field has the discounted amount already.

Reply:
@atchuthan My concern is with the Avatax API call sending wrong amounts. Can you double check that?

Reply:
@dreispt Yes, our customer is using this modified code for few months, and they haven't noticed any problem with the transactions

P.S: Sorry for the late reply.  Is it possible to reopen this PR?